scripts/upgrade_admin_server.sh

```diff
@@ -91,8 +99,13 @@ n.target_platform = 'suse-12.2'
 n.provisioner.default_os = 'suse-12.2'
 n.save"
 
-    # Signalize that the upgrade correctly ended
-    echo "12.2" >> $UPGRADEDIR/admin-server-upgraded-ok
+    ret=$?
+    if [ $ret != 0 ]; then
+        report_failure($ret, "Setting the platform to suse-12.2 has failed.")
```

Comment:
Not sure if it makes a bit difference here. But shouldn't we acutally exit here, instead of just skipping the ```admin-server-upgraded-ok``` step